framework/branch-key-store.md

```diff
@@ -109,6 +156,42 @@ If the [AWS KMS Configuration](#aws-kms-configuration) designates MRK ARN compat
 then two ARNs are compatible if they are equal in all parts other than the region.
 That is, they are compatible if [AWS KMS MRK Match for Decrypt](aws-kms/aws-kms-mrk-match-for-decrypt.md#implementation) returns true.
 
+If the [AWS KMS Configuration](#aws-kms-configuration) is Discovery or MRDiscovery,
+no comparison is ever made between ARNs.
+
+#### Discovery
+
+Discovery takes no additional information.
+
+The Keystore can use ANY KMS Key ARN already
+persisted to the backing DynamoDB table,
```

Comment:
Issue/Suggestion:
(on behalf of @josecorella ):
```suggestion
persisted on the Key Store table,
```

@josecorella what is "the Keystore table"?.
That is not defined, and does not exist.
Where as a DynamoDB Table is an established concept outside of the Keystore Spec.

Hence, I believe "persisted to the backing DynamoDB table" is 
easier to understand.

Reply:
We have brought up the idea of not needing to store in Dynamo. There is even a feature request for this. So I suggest we start moving to some more ambiguous terms.

Reply:
Than give me an ambiguous term.

Is the ambiguous term "Keystore Table"?

Reply:
Resolving. 
When we introduce a different persistence medium, we can revise this then.